src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -1716,6 +1724,30 @@ double ExtendedDefaultPolicy::DetermineMultiplier()
         JITDUMP("\nPrejit root candidate has arg that feeds a conditional.  Multiplier increased to %g.", multiplier);
     }
 
+    if (m_ArgUnboxExact > 0)
```

Comment:
@AndyAyersMS I think you can also rely on existing `m_ArgIsBoxedAtCallsite` here, so if an arg is boxed on the callsite and unboxed in the callee - it should be quite profitable